cmd/admin-replicate-update.go

```diff
@@ -39,9 +39,9 @@ var adminReplicateUpdateFlags = []cli.Flag{
 		Usage: "endpoint for the site",
 	},
 	cli.StringFlag{
-		Name:  "sync",
-		Usage: "enable synchronous replication for this target, valid values are ['enable', 'disable'].",
-		Value: "disable",
+		Name:  "mode",
```

Comment:
I think you can add 'sync' to Aliases field, to keep backward compatibility

Reply:
well, the field changed to take on different values - so will have to validate this separately